### ISSUE TITLE
Fill Plates

### ISSUE THREAD

**Issue by Nebukadneza** [tshort/dactyl-keyboard#19]
The plate files generatable by dactyl.clj (once you add a mirrored left variant) merely consist of the wall outlines, not the full plate. They essentially are a brim of plate around the (inner) perimeter of the keybord. If i got it wrong what they should be, please excuse my issue. On the other hand, it’d be great if it was possible to generate »full« plates, that are usable as-is, for the keyboard. Sadly, my openscad knowledge is not as good, so i didn’t succeed yet in making them »full«. If i succeed, i’ll open an PR — until then, i’d be grateful for any tips that more seasoned users might have ^_^.

Thank & Best,
-Dario

**Comment by tshort** [comment]
Yes, the bottom plates have the outline and the screw holes. As I recall, I needed to do some manual editing of the dxf files to remove unneeded parts. I couldn't figure out how to fully automate it.

**Comment by Nebukadneza** [comment]
Yes, i also couldn’t find out how to fill them … `hull()` as is the convex hull isn’t quite right (produces a too large plate …), and the other similar operation, `minowski()` is also not what we want.

One thing i took a while to understand is that openscad can spout out a 2d DXF of the plate, which you can then extrude in the CAD of your choice. All that’s left is to add a generation procedure for the left side of the plate.

Since i think it’d help others, i’ll probably PR generation of both sides of the plates, with a small 
`README.md` hint reminding other users of the DXF+CAD way … if that’s ok?

Thanks and Best,
-Dario

**Comment by maxkorp** [comment]
There is an orientation that works to slice the plate, hull the slices, and union those hulls, but I was still tuning it for performance and then lost my changes. It's definitely workable though
